docs/fundamentals/jcloud/advanced.md

```diff
@@ -73,6 +73,101 @@ executors:
           type: efs
 ```
 
+### GPU support
+
+We have GPU support in JCloud. We have `2` kind of GPU usages, `shared` or `dedicated`.
+
+```{note}
+Due to enable GPU support may require to cluster to dynamically provision new GPU nodes. It may take extra 2-3 mins till all flows are running in the cluster.
+```
+
+#### Shared GPU
+
+Shared GPU is to enable time-slicing feature, which is used to allow workloads that land on oversubscribed GPUs to interleave with one another. In that case, executor may share a GPU with others(10 executor at maximum).
+
+```yaml
+jtype: Flow
+protocol: http
+executors:
+- name: sb0
+  uses: jinahub+docker://CustomExecutor
+  jcloud:
+      resources:
+        gpu: shared
+```
+
+```{note}
+Please notice, nothing special is done to isolate workloads that are granted replicas from the same underlying GPU, and each workload has access to the GPU memory and runs in the same fault-domain as of all the others (meaning if one workload crashes, they all do).  
+```
```

Comment:
```suggestion
```{caution}
There are no special previsions in place to isolate replicas that run on the same underlying GPU. Each workload has access to the GPU memory and runs in the same fault-domain as of all the others. Therefore, if one workload crashes, they all do.  
```
```